ingestion-data/staging/collections/MODIS_Terra_L3_NDVI_16Day.json

```
@@ -0,0 +1,77 @@
+{
+  "assets": {},
+  "id": "MODIS_Terra_L3_NDVI_16Day",
+  "dashboard:is_periodic": true,
+  "dashboard:time_density": "day",
+  "dashboard:time_interval": "P1D",
+  "data_type": "cog",
+  "description": "The Vegetation Index (L3, 16-Day) layer is created from the Terra Moderate Resolution Imaging Spectroradiometer (MODIS) Vegetation Indices (MOD13Q1) data which are generated every 16 days at 250 meter (m) spatial resolution as a Level 3 product. The MOD13Q1 product provides the Normalized Difference Vegetation Index (NDVI) which is referred to as the continuity index to the existing National Oceanic and Atmospheric Administration-Advanced Very High Resolution Radiometer (NOAA-AVHRR) derived NDVI. The algorithm chooses the best available pixel value from all the acquisitions from the 16-day period. The criteria used is low clouds, low view angle, and the highest NDVI value. The MODIS Normalized Difference Vegetation Index (NDVI) complements NOAA's Advanced Very High Resolution Radiometer (AVHRR) NDVI products and provides continuity for time series historical applications. The MODIS NDVI product is computed from surface reflectances corrected for molecular scattering, ozone absorption, and aerosols. References: MOD13Q1 doi:10.5067/MODIS/MOD13Q1.061",
+  "extent": {
+    "spatial": {
+      "bbox": [
+        [
+          -180,
+          -90,
+          180,
+          90
+        ]
+      ]
+    },
+    "temporal": {
+      "interval": [
+        [
+          "2000-03-05T00:00:00Z",
+          "2025-06-30T00:00:00Z"
+        ]
+      ]
+    }
+  },
+  "is_periodic": true,
+  "item_assets": {
+    "cog_default": {
+      "description": "Vegetation Index (L3, 16-Day)",
+      "roles": [
+        "data",
+        "layer"
+      ],
+      "title": "MODIS Terra NDVI",
+      "type": "image/tiff; application=geotiff; profile=cloud-optimized"
+    }
+  },
+  "license": "CC0-1.0",
+  "links": [
+    {
+      "href": "https://gibs.earthdata.nasa.gov/wmts/epsg3857/best/wmts.cgi",
+      "rel": "wmts",
+      "type": "image/png",
+      "title": "Visualized through a WMTS",
+      "href:servers": [
+        "https://gibs-a.earthdata.nasa.gov/wmts/epsg3857/best/wmts.cgi",
+        "https://gibs-b.earthdata.nasa.gov/wmts/epsg3857/best/wmts.cgi"
+      ],
+      "wmts:dimensions": [
+        "default"
+      ],
+      "wmts:layers": [
+        "MODIS_Terra_L3_NDVI_16Day"
+      ]
+    }
+  ],
+  "product_level": "L3",
+  "providers": [],
```

```suggestion
  "providers": [
      {
      "name": "NASA Global Imagery Browse Services (GIBS)",
      "roles": [
        "host"
      ],
      "url": "https://nasa-gibs.github.io/gibs-api-docs/gis-usage/"
    }
  ],
```

perhaps more of these provider types are also applicable? 
licensor, producer, processor 
(ref: https://github.com/radiantearth/stac-spec/blob/master/collection-spec/collection-spec.md#provider-object)

---

I think that we should decide what all we want to include as providers. We will likely be pulling more and more data from GIBS and potentially other WMTS layers from ArcGIS so it may behoove us to set some kind of a standard.